Lib/test/test_logging.py

```diff
@@ -3532,7 +3532,8 @@ def test_baseconfig(self):
         self.assertEqual(bc.convert('cfg://nest2[1][1]'), 'm')
         self.assertEqual(bc.convert('cfg://adict.d'), 'e')
         self.assertEqual(bc.convert('cfg://adict[f]'), 3)
-        self.assertEqual(bc.convert('cfg://adict[alpha nummeric 1 with spaces]'), 5)
+        self.assertEqual(bc.convert('cfg://adict[alpha nmmeric 1 with spaces]'), 5)
+        self.assertEqual(bc.convert('cfg://adict[alpha nmmeric 1 with spaces]'), 9)
```

Comment:
Should it be `alpha numeric` instead of `alpha nmmeric`?

Reply:
Yes.
I have addressed this. 
Sorry about it.